AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -12,6 +12,7 @@ forum.duolingo.com#?#.ReactModal__Overlay:has(> div.ReactModal__Content h2[class
 goo-net.com###floatingNaviCarlease
 aspor.com.tr###contactLink
 cnet.com###video-rectangle
+yay.space##.AnonymousCallListPopup
```

Comment:
Not annoying.

Reply:
IMO `##div[class^="Home__app-download-"]` too.

Reply:
@SKEIDs Please leave `yay.space##.AppDownloadAds` only.

Reply:
I see